Source/PropertyTools.Wpf/PropertyGrid/PropertyGrid.cs

```diff
@@ -1370,7 +1371,8 @@ private void AddPropertyPanel(Panel panel, PropertyItem pi, object instance, Tab
                         propertyControl.Style = this.ValidationErrorStyle;
                     }
 
-                    errorControl = CreateErrorControl(pi, instance, tab);
+                    PropertyControlFactoryOptions options = new PropertyControlFactoryOptions { ValidationErrorTemplate = this.ValidationErrorTemplate };                    

```

Comment:
move this code into the ´PropertyGrid.CreateErrorControl´ method?
or remove the ´PropertyGrid.CreateErrorControl´ method